packages/electron/src/renderer/addBacktraceElectron.ts

```diff
@@ -0,0 +1,53 @@
+import { BacktraceCoreClientBuilder } from '@backtrace-labs/sdk-core';
+import { IpcEvents } from '../common/ipc/IpcEvents';
+import { SyncData } from '../common/models/SyncData';
+import { getIpcRpc } from './ipc/getIpcRpc';
+import { getIpcTransport } from './ipc/getIpcTransport';
+import { ConstSessionProvider } from './modules/ConstSessionProvider';
+import { IpcAsyncSessionProvider } from './modules/IpcAsyncSessionProvider';
+import { IpcBreadcrumbsStorage } from './modules/IpcBreadcrumbsStorage';
+import { IpcReportSubmission } from './modules/IpcReportSubmission';
+import { IpcRequestHandler } from './modules/IpcRequestHandler';
+import { IpcSummedMetricsQueue } from './modules/IpcSummedMetricsQueue';
+import { StubMetricsQueue } from './modules/StubMetricsQueue';
+
+export interface AddBacktraceElectronOptions {
+    readonly synchronous?: boolean;
+}
+
+export function addBacktraceElectron<T extends BacktraceCoreClientBuilder>(
```

Comment:
Can we add documentation what it does any why someone should use it?

Reply:
I'll make sure to add that in the future.

As stated in the PR description:
>This release is NOT production ready - docs need to be written, READMEs, better import behavior, etc.